CONTRIBUTING.md

```diff
@@ -14,6 +14,7 @@ Before you start, install the following dependencies.
 
 * [Visual Studio Code](https://code.visualstudio.com/Download)
 * [NodeJS and NPM](https://nodejs.org/)
+* [Typescript](https://www.typescriptlang.org/)
```

Comment:
FYI, this shouldn't require any manual action--typescript is included as an npm dependency. If contributors *do* need a global install for some reason, they should install with `npm install -g typescript`.